src/utils/constants.py

```diff
@@ -0,0 +1,53 @@
+"""this module contains constants used in the rest of this library"""
+
+from dataclasses import dataclass
+
+from beartype.typing import Literal
+
+ObjectType = Literal["row", "database", "workspace"]
```

Comment:
Really such a shame there's no way to pull this from the openapi spec.

Reply:
this will be addressed in a future PR